apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -526,6 +526,10 @@ internal async Task Process(long byteCount, string orderingKey, Func<Task> fn)
                             // Add to stats for this element.
                             _byteCount += byteCount;
                             _elementCount += 1;
+                            // If there's no ordering-key then the user callback function can always immediately be executed
+                            // because there's no ordering constraint to meet.
+                            // If there is an ordering-key then the user callback function must be exeuted sequentially per
```

Comment:
Nit: exeuted => executed

(I realize this was just a deliberate typo to check that I'm actually reviewing... ;)

Reply:
Done :)